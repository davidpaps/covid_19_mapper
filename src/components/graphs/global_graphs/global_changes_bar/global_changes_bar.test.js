import * as React from "react";
import { shallow } from "enzyme";
import GlobalChangesBar from "./global_changes_bar";

describe("Global Changes Bar", () => {
  let wrapper;

  beforeEach(() => (wrapper = shallow(<GlobalChangesBar data={[]} />)));

  it("should render a <div />", () => {
    expect(wrapper.find("div").length).toEqual(2);
  });

  it("should render a <Bar />", () => {
    expect(wrapper.find("Bar").length).toEqual(1);
  });

  it("renders correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
