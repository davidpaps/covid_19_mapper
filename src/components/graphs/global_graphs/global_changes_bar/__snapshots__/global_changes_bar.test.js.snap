// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Global Changes Bar renders correctly 1`] = `ShallowWrapper {}`;
